cprover_bindings/Cargo.toml

```diff
@@ -3,7 +3,7 @@
 
 [package]
 name = "cprover_bindings"
-version = "0.0.0"
+version = "0.4.0"
 edition = "2018"
```

Comment:
Do we want to upgrade to Rust 2021 here? Same goes for library packages.

Reply:
We could look into that, yeah. Not going to do it for this PR though.